keylime/tpm2.py

```diff
@@ -404,6 +448,30 @@ def __get_pub_ek(self): # assumes that owner_pw is correct at this point
         
         self._set_tpm_metadata('ek', ek)
 
+    def __get_pub_ak(self):
```

Comment:
Could we add a docstring here , just a little on what the function achieves, args processed and expected return:

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html